ios/FluentUI/ActivityIndicator/ActivityIndicator.swift

```diff
@@ -173,25 +132,24 @@ public struct ActivityIndicator: View {
     private let finalAnimationRotationAngle: Double = 360.0
 }
 
-/// Properties available to customize the state of the activity indicator
-class MSFActivityIndicatorStateImpl: NSObject, ObservableObject, MSFActivityIndicatorState {
+/// Properties available to customize the state of the Activity Indicator
+class MSFActivityIndicatorStateImpl: NSObject, ObservableObject, ControlConfiguration, MSFActivityIndicatorState {
+    /// Design token set for this control, to use in place of the control's default Fluent tokens.
+    @Published public var overrideTokens: ActivityIndicatorTokens?
+
+    /// On-demand default token set.
+    var defaultTokens: ActivityIndicatorTokens
     @Published var color: UIColor?
     @Published var isAnimating: Bool = false
     @Published var hidesWhenStopped: Bool = true
 
-    let tokens: MSFActivityIndicatorTokens
-
-    var size: MSFActivityIndicatorSize {
-        get {
-            return tokens.size
-        }
-
-        set {
-            tokens.size = newValue
-        }
-    }
+    // TODO: Setting this property needs to cause the tokens to update
```

Comment:
We need to track the issue of updating the design tokens when the size property changes in the state object. The same applies for other controls that use either "style" or "size" enumerations that affect which tokens are used in the control.

Reply:
+1

Got a local branch with this working; hoping to have a PR out soon.